app/src/main/res/layout/home_fragment.xml

```diff
@@ -46,6 +46,9 @@
       android:id="@+id/home_recycler_view"
       android:layout_width="match_parent"
       android:layout_height="match_parent"
+      android:layout_marginEnd="24dp"
```

Comment:
Suggestions: These margins will disturb your implementation of "All Topics" text too. So either you can change this PR or you can make change in #315

Reply:
I have just set left and right margins for this and top bottom margin is set to "All Topics" in [#315](https://github.com/oppia/oppia-android/pull/315) so it will not disturb any views here.